frontend/src/modules/Survey/Components/StepBegin.tsx

```diff
@@ -1,4 +1,6 @@
 import React, { useState } from 'react'
+import { IconButton, Box } from '@mui/material'
```

Comment:
Nice use of the mui icons! it makes the code much simpler and the  look more consistent